### PR TITLE
Syndicate drip part 1

### PR DESCRIPTION
**Pull request by 1glitchycent**
## About The Pull Request
Adds a Gorlex Marauders officer uniform, intended to be used for captains on most Syndicate vessels

## Why It's Good For The Game
The Syndicate is lacking in job uniforms, this will help remedy that

## Changelog
:cl:
add: Added a jumpsuit, coat and hat for syndicate captains.
/:cl:

### CODE REVIEW

**Comment by 1glitchycent** [comment]
pictures 
image
image
image

File: whitesands/code/modules/clothing/suits/coats.dm [hunk 1]
```diff
@@ -64,3 +64,14 @@
 	desc = "An armored coat worn for special occasions. This one is still dyed in original TerraGov green."
 	icon_state = "coat_terragov"
 	item_state = "coat_terragov"
+
+/obj/item/clothing/suit/toggle/solgov
```
Comment (MarkSuckerberg) [inline whitesands/code/modules/clothing/suits/coats.dm]: Needs new path

File: whitesands/code/modules/clothing/head/misc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/obj/item/clothing/head/aclfcap
+	name = "2nd Battlegroup peaked cap"
+	desc = "A cap worn by officers of the Gorlex Marauders 2nd Battlegroup, or as they say, the ACLF."
+	icon = 'whitesands/icons/obj/clothing/hats.dmi'
+	mob_overlay_icon = 'whitesands/icons/mob/clothing/head.dmi'
+	icon_state = "aclfcap"
+	item_state = "aclfcap"
+	flags_inv = 0
+	armor = list("melee" = 25, "bullet" = 15, "laser" = 25, "energy" = 35, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
+	strip_delay = 60
```
Comment (MarkSuckerberg) [inline whitesands/code/modules/clothing/head/misc]: File needs to end with .dm and be included in the DME

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by triplezeta** [comment]
this should be ready for re-review

nice sprites btw

**Comment by triplezeta** [comment]
do you think you could get some screenshots of them in-game?

**Pull request merged**